modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/liferay.d.ts

```diff
@@ -238,6 +238,35 @@ declare module Liferay {
 	}
 
 	namespace Util {
+		namespace Cookie {
+
+			/**
+			 * Object with cookie consent types as keys, for use in {@link Cookie.set}
+			 */
+			export const TYPES: Object;
+
+			/* Returns the stored value of a cookie, undefined if not present */
+			export function get(name: string): string | undefined;
+
+			/* Sets a cookie of a specific type if user has consented */
+			export function set(
+				name: string,
+				value: string,
+				type: string,
+				options: {
+					'domain'?: string;
+					'expires'?: string;
+					'max-age'?: string;
+					'path'?: string;
+					'samesite'?: string;
+					'secure'?: boolean;
```

Comment:
It looks weird having these be strings, i mean the  keys (`'domain'`, `'expires'`) being wrapped with `''`. Does it not allow you to add the `?` if it's not a string?

Reply:
Ah i see 👍